optuna/dashboard.py

```diff
@@ -1,6 +1,10 @@
 import collections
 import threading
 import time
+from typing import Any  # NOQA
+from typing import Dict  # NOQA
+from typing import List  # NOQA
+from typing import Optional  # NOQA
```

Comment:
We can just remove `# NOQA`. Could you remove them?

Reply:
Seems to apply to the import of `Any` in `optuna/integration/__init__.py` as well. 🙂

Reply:
Thank you for your review.
I fixed the code as you advised.